### ISSUE TITLE
1.20 support

### ISSUE THREAD

**Issue by LightslicerGP** [TheAlienDoctor/minecraft-resource-pack-converter#14]
title, i mean its been a while since the version was released so it should be done

**Comment by TheAlienDoctor** [comment]
This is in the works, but due to myself being busy IRL and with other projects its taking longer than expected. Thanks for your patience.

**Comment by LightslicerGP** [comment]
No problem!

**Comment by LightslicerGP** [comment]
Update 7/26/23 at 12:16pm:

![image](https://github.com/TheAlienDoctor/minecraft-resource-pack-converter/assets/63020072/6f2d4a48-b482-4d1d-9e22-1b9f34cfcf63)

**Comment by TheAlienDoctor** [comment]
All I need to do is add the entities, and possibly some other misc stuff then I should be done 👍
I'm also gonna add an option in the options file that lets you change the Bedrock min_engine_version and Java format_version

**Comment by TheAlienDoctor** [comment]
Oops didn't mean to close the issue 😳

**Comment by TheAlienDoctor** [comment]
Update: I'm hoping to have this ready by the end of the week. I was away from home last week so didn't have a chance to work on this, and then the Friday before that I lost some important data on my computer which has led me to needing to rebuild all of that since I got back, but I should be back on track for updating this again now :D

**Comment by TheAlienDoctor** [comment]
Ad you might have noticed, didn't get a chance to finish this.

If you install AutoIt and run the script from its source it will be able to convert 1.20 blocks and items.

I have finished adding all the conversion variables for the other entities and misc stuff. Although I forgot to push those changes 🙄

I need to finish a couple of update related features, the main being the ability to choose the pack version via the options file. Once this is done I'll release the update.

I'd say about a week more but I don't want to put a time on it in case plans change lol.

Thanks for the patience, we're in the endgame now!

-Alien

**Comment by LightslicerGP** [comment]
Why not compile it as is now as a major release, then release smaller things for the bug fixes so atleast people have a 1.20 version to work with?

**Comment by TheAlienDoctor** [comment]
Fair enough, sorry had a busy day yesterday so wasn't thinking entirely straight. I think I'll publish a beta.
-Alien

**Comment by LightslicerGP** [comment]
Lmao it's no worries, thanks!

**Comment by TheAlienDoctor** [comment]
V1.4.0-Beta1: https://github.com/TheAlienDoctor/minecraft-resource-pack-converter/releases/tag/V1.4.0-Beta1
